fdbserver/storageserver.actor.cpp

```diff
@@ -2196,6 +2244,9 @@ ACTOR Future<Void> fetchKeys( StorageServer *data, AddingShard* shard ) {
 		data->counters.fetchExecutingMS += 1000*(now() - executeStart);
 
 		TraceEvent(SevDebug, interval.end(), data->thisServerID);
+
+		reportHistogram();
```

Comment:
You  shouldn't invoke reportHistogram directly.  It's registered for automatic reporting when you instantiate it.

Reply:
(oops; misread this)

Reply:
Yup, that is just a handy lambda prevent me rewriting same piece of code in throw clause.